docs/service-catalog/05-02-catalog.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Catalog sub-chart
+type: Configuration
+---
+
+To configure the Catalog sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```

Comment:
```suggestion
To configure the `Catalog` sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```
not sure if it should be `Catalog` or `catalog` as usually chart names are not capitalized (unless I guess it's a proper name, like Service Catalog which is always capitalized)🤔

Reply:
Discussed